guides/workspace-settings/labels.md

```diff
@@ -1,14 +1,25 @@
 # Labels
-Labels are great for your team's administration to organize certain people. 
+Labels are great for your team's administration to organize certain people. You can use them to label staff from their timezone, perks, or status.
 
-Some ideas may be:
-- Timezone
-- Partnerships/Development/Support/Events/Moderation Team
-- Perks
-- Up for Promotion/Demotion
+:::caution PERMISSION REQUIRED
+To proceed, please note that you need the following permissions: **Workspace Admin**
+:::
 
-![labels](/img/labels.PNG)
+## Creating Labels
+To create a label, go to Settings > Labels and click "Add Label."
```

Comment:
Labels can also be created from the add label UI

Reply:
✅ Done.